main.go

```diff
@@ -13,29 +14,28 @@ func main() {
 		help()
 		return
 	}
-	var err error
-	switch flag.Arg(0) {
-	case "new":
-		err = commands.New()
-	case "login":
-		err = commands.Login(flag.Arg(1))
-	case "edit":
-		err = commands.Edit(flag.Arg(1))
-	case "worked":
-		err = commands.Worked(flag.Arg(1))
-	case "link":
-		err = commands.Link(flag.Arg(1))
-	case "editor":
-		err = commands.Editor(flag.Arg(1))
-	default:
-		err = commands.Query(flag.Arg(0))
-	}
-	if err != nil {
-		fmt.Println(err)
-		os.Exit(1)
+
+	if c, ok := commands.Commands[flag.Arg(0)]; !ok {
+		commands.Query(flag.Arg(0))
+	} else {
+		if err := c.F(flag.Args()); err != nil {
+			fmt.Println(err)
+			os.Exit(1)
+		}
 	}
 }
 
 func help() {
-	fmt.Println("Usage: borg \"your question\"")
+	fmt.Print("\033[4mUsage:\033[0m\n\n")
```

Comment:
@crufter @anas10 is this for colour highlighting or something similar? I am referring to `\033[4m`. If so, I am sure you can abstract it into a method that wraps a string around a specific colour?